webpack.config.js

```diff
@@ -15,15 +15,17 @@ module.exports = {
     filename: 'js/[name].js'
   },
   resolve: {
-    extensions: ['.js', '.vue', '.json'],
+    extensions: ['.js', '.vue', '.json', '.css', '.scss'],
     alias: {
       '@': path.join(__dirname, 'src')
     }
   },
   module: {
     rules: [
       { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader' },
-      { test: /\.vue$/, loader: 'vue-loader' }
+      { test: /\.vue$/, loader: 'vue-loader' },
+      { test: /\.css$/, use: [ 'vue-style-loader', 'css-loader' ] },
+      { test: /\.scss$/, use: [ 'vue-style-loader', 'css-loader', 'sass-loader' ] }
```

Comment:
I feel like this can be simplified but not 100% sure about that and it works so well done.